### PR TITLE
FAKEHBASE-4. Implement HBaseAdmin.getTableRegions()

### PR DESCRIPTION
**Pull request by kryzthov**

### CODE REVIEW

**Comment by kryzthov** [comment]
@robotoer could you have a look at this? thanks in advance

**Comment by kryzthov** [comment]
@robotoer ping?

File: src/main/scala/org/kiji/testing/fakehtable/FakeHBase.scala [hunk 1]
```diff
@@ -29,16 +35,49 @@ import org.apache.hadoop.hbase.util.Bytes
...
-import net.sf.cglib.proxy.MethodProxy
-import org.kiji.schema.impl.HBaseInterface
+import org.apache.hadoop.hbase.TableNotDisabledException
+import org.apache.hadoop.hbase.TableNotFoundException
 import org.kiji.schema.impl.HBaseAdminFactory
+import org.kiji.schema.impl.HBaseInterface
 
 // -------------------------------------------------------------------------------------------------
 
+object MD5Space {
```
Comment (robotoer) [inline src/main/scala/org/kiji/testing/fakehtable/FakeHBase.scala]: Any reason why this is not in a separate file? While I understand that this is very related to FakeHBase, I don't like having multiple classes in the same file if possible.

**Comment by robotoer** [comment]
Just one comment, looks good otherwise.

**Comment by kryzthov** [comment]
@robotoer Split MD5Space into its own source file.

**Pull request closed without merging**